### PR TITLE
enh(auth): open id connect

### PR DESCRIPTION
**Pull request by lpinsivy**
Documentation for https://github.com/centreon/centreon/pull/9025

## Target serie

- [ ] 19.04.x
- [ ] 19.10.x
- [ ] 20.04.x
- [x] 20.10.x (master)

### CODE REVIEW

File: fr/administration/parameters/centreon-ui.md [hunk 1]
```diff
@@ -99,25 +99,41 @@ indique l'expression rationnelle (pattern) de recherche pour l'utilisateur.
...
-- Le champ **Identifiant client Keycloak** indique l'identifiant du client Keycloak.
-- Le champ **Secret client Keycloak** indique le secret du client Keycloak.
-
-Pour plus d'informations, se référer à la documentation Keycloak standard.
+## OpenId Connect
+
+Centreon est compatible avec l'authentification OAuth 2.0 / OpenId Connect.
+
+Il est possible d'utiliser un fournisseur d'identité (IdP) tel que Keycloak, LemonLDAP::NG ou tout autre IdP compatible
+avec le flux de code d'autorisation (uthorization code) de type d'autorisation (grant type).
```
Comment (sc979) [inline fr/administration/parameters/centreon-ui.md:107]:
```suggestion
avec le flux de code d'autorisation (authorization code) de type d'autorisation (grant type).
```

File: en/administration/parameters/centreon-ui.md [hunk 2]
```diff
@@ -92,26 +92,39 @@ the username.
...
-- **Keycloak Client Realm** field defined the Keycloack realm name.
-- **Keycloak Client ID** field defined the Keycloack client ID.
-- **Keycloak Client Secret** field defined the Keycloack client secret.
-
-For more information, please refer to standard Keycloak documentation.
+## OpenId Connect
+
+Centreon is compatible with OAuth 2.0 / OpenId Connect authentication.
+
+It is possible to use Identity Provider (IdP) such as Keycloak, LemonLDAP::NG or other IdP which are compatible with
```
Comment (sc979) [inline en/administration/parameters/centreon-ui.md:99]:
```suggestion
Usage of Identity Provider (IdP) is available, such as Keycloak, LemonLDAP::NG or other IdP which are compatible with
```

File: en/administration/parameters/centreon-ui.md [hunk 3]
```diff
@@ -92,26 +92,39 @@ the username.
...
+- **Scope** field defines the scope of the IdP, for example `openid`. Separate scope by space.
+- **Redirect Url** field defines the redirect URL after login to access to your Centreon server, for example
+  `htt://192.168.0.1/centreon/index.php`.
+- **Client ID** field defines the Client ID.
+- **Client Secret** field defines the Client secret.
+- **Disable SSL verify peer** field allows to disable SSL peer validation, should only used for test.
+
+![image](../../assets/administration/openid-connect-configuration.png)
+
+> Depending on the identity provider, it is necessary to enter several scopes, this is indicated in the provider's
```
Comment (sc979) [inline en/administration/parameters/centreon-ui.md:122]:
```suggestion
> Depending on the identity provider, it is necessary to enter several scopes. This is indicated in the provider's
```

File: en/administration/parameters/centreon-ui.md [hunk 4]
```diff
@@ -92,26 +92,39 @@ the username.
...
+  reverse proxy). The trusted clients are separated by comas.
+- **Blacklist client addresses** field indicates which are the IP/DNS rejected.
+- **Base Url** field defines the IdP base URL for OpenId Connect endpoints (mandatory).
+- **Authorization Endpoint** field defines the authorization endpoint, for example `/authorize` (mandatory).
+- **Token Endpoint** field defines the token endpoint, for example `/token`(mandatory).
+- **Introspection Token Endpoint** field defines the introspection token endpoint, for example `/introspect` (mandatory).
+- **User Information Endpoint** field defines the user information endpoint, for example `/userinfo`.
+- **Scope** field defines the scope of the IdP, for example `openid`. Separate scope by space.
+- **Redirect Url** field defines the redirect URL after login to access to your Centreon server, for example
+  `htt://192.168.0.1/centreon/index.php`.
```
Comment (cgagnaire) [inline en/administration/parameters/centreon-ui.md:115]:
```suggestion
  `http://192.168.0.1/centreon/index.php`.
```

File: fr/administration/parameters/centreon-ui.md [hunk 5]
```diff
@@ -99,25 +99,41 @@ indique l'expression rationnelle (pattern) de recherche pour l'utilisateur.
...
+  l'adresse du reverse proxy). Chaque client de confiance est séparé par une virgule.
+- Le champ **Blacklist client addresses** indique quelles sont les adresses IP/DNS des clients qui seront refusés.
+- Le champ **Base Url** définit l'URL de base de l'IdP pour les points de terminaison OpenId Connect (obligatoire).
+- Le champ **Authorization Endpoint** définit le point de terminaison d'autorisation, par exemple `/authorize` (obligatoire).
+- Le champ **Token Endpoint** définit le point de terminaison du jeton, par exemple `/token` (obligatoire).
+- Le champ **Introspection Token Endpoint** définit le point de terminaison du jeton d'introspection, par exemple `/introspect` (obligatoire).
+- Le champ **User Information Endpoint** définit le point de terminaison des informations utilisateur, par exemple `/userinfo`.
+- Le champ **Scope** définit la portée de l'IdP, par exemple «openid». Portée séparée par espace.
+- La case **Redirect Url** définit l'URL de redirection après connexion pour accéder à votre serveur Centreon, par exemple
+  `htt://192.168.0.1/centreon/index.php`.
```
Comment (cgagnaire) [inline fr/administration/parameters/centreon-ui.md:122]:
```suggestion
  `http://192.168.0.1/centreon/index.php`.
```

**Review by cgagnaire — APPROVED** [review]
Please try to respect the 80 characters width rule.

**Pull request merged**